src/lib.rs

```diff
@@ -65,7 +65,7 @@
 //!# -> Result<(), QdrantError> {
 //! let points = vec![
 //!     PointStruct::new(
-//!         42,                 // Uniqe piont ID
+//!         42,                 // Uniqe point ID
```

Comment:
Two fixes in one go :smiley: 

```suggestion
//!         42,                 // Unique point ID
```